docs/Model/WebPageAclMappingModel.md

```diff
@@ -0,0 +1,34 @@
+<!-- generated file with tool "Kentico.Xperience.UMT.DocUtils" - edited through template "UmtModel.cshtml" -->
+## WebPageAclMappingModel
```

Comment:
The WebPageAclMappingModel, should not be public. It is something we internally use for materialization of inherited ACLs defined in WebPageAcl table. The system should be responsible for creation of these mappings, not user.